src/EditorFeatures/Core.Wpf/Microsoft.CodeAnalysis.EditorFeatures.Wpf.csproj

```diff
@@ -78,4 +78,14 @@
   <ItemGroup>
     <EmbeddedResource Update="EditorFeaturesWpfResources.resx" GenerateSource="true" />
   </ItemGroup>
+
+
+  <!-- TODO: Remove once we consume an SDK with the fix from https://github.com/dotnet/wpf/pull/6799 -->
+  <Target Name="RemoveDuplicateAnalyzers" BeforeTargets="CoreCompile">
+    <ItemGroup>
+        <FilteredAnalyzer Include="@(Analyzer->Distinct())" />
+        <Analyzer Remove="@(Analyzer)" />
+        <Analyzer Include="@(FilteredAnalyzer)" />
+    </ItemGroup>
+  </Target>
```

Comment:
I think the root source of duplication is:

https://github.com/dotnet/wpf/blob/c5b6632f5f80f07869e1ae4ec3fcd976f18c2083/src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/GenerateTemporaryTargetAssembly.cs#L266-L267

This produces a whole `ItemGroup` in the wpftmp project:

![image](https://user-images.githubusercontent.com/31348972/184083044-c07b7e37-00d3-4bcf-8193-9823566ef43f.png)